pgrx/src/memcx.rs

```diff
@@ -68,6 +68,16 @@ impl<'mcx> MemCx<'mcx> {
     }
 }
 
+#[derive(Debug)]
+pub struct OutOfMemory {
+    _reserve: (),
+}
+impl OutOfMemory {
+    pub fn new() -> OutOfMemory {
+        OutOfMemory { _reserve: () }
+    }
+}
+
```

Comment:
Is it worth it to have this be an actual Error and store the size it tried to allocate that it couldn't, and have that reported in its Display impl?